backend/src/handlers/transfer_operations.py

```
@@ -35,325 +41,339 @@ def get_transaction_by_id(transaction_id: uuid.UUID) -> Optional[Transaction]:
+        try:
+            from datetime import datetime
+            start_date_ts = int(start_date_param)
+            end_date_ts = int(end_date_param)
+            
+            # Convert to datetime objects for logging
+            start_date = datetime.fromtimestamp(start_date_ts / 1000)
+            end_date = datetime.fromtimestamp(end_date_ts / 1000)
+            
+            logger.info(f"Using date range: {start_date.isoformat()} to {end_date.isoformat()}")
+        except (ValueError, TypeError) as e:
+            logger.error(f"Invalid date format: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)